descriptors/ModuleDescriptor-template.json

```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "circulation",
-      "version": "4.1",
+      "version": "4.2",
```

Comment:
If a service point is required for check out, then I believe this is a interface compatibility breaking changes.

If so we need to coordinate with the relevant parts of the UI to ensure coherent version resolution and that the UI continues to function (we may need to accept some breakage of folio-testing following the merge).

Reply:
You are correct, this should be a 5.0